packages/react/src/Form/InputNumber/NumberInput.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  useId,
+  ComponentPropsWithoutRef,
+  ComponentPropsWithRef,
+  ReactNode,
+} from "react";
+import { Field, HelpMessage, FieldInput, useInputClassModifier } from "../core";
+import "@axa-fr/design-system-css/dist/common/reboot.scss";
+import "@axa-fr/design-system-css/dist/common/grid.scss";
+import "@axa-fr/design-system-css/dist/Form/core/FormCore.agent.scss";
+
+import { Number } from "./Number";
+
+type Props = ComponentPropsWithoutRef<typeof Field> &
+  ComponentPropsWithRef<typeof Number> & {
+    helpMessage?: ReactNode;
+    children?: ReactNode;
+  };
+
+export const NumberInput = ({
+  message,
+  children,
+  helpMessage,
+  id,
+  label,
+  classNameContainerLabel,
+  classNameContainerInput,
+  forceDisplayMessage,
+  messageType,
+  isVisible,
+  className,
+  disabled = false,
+  classModifier = "",
+  ...otherProps
+}: Props) => {
+  const inputUseId = useId();
+  const inputId = id ?? inputUseId;
```

Comment:
```suggestion
  const inputId = id || inputUseId;
```
Grâce à ça, si `id` est une string vide on va utiliser `inputUseId`.